CIP-_/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: "?"
+Title: Optimized Coin Selection Algorithms for Cardano
+Category: Wallets
+Authors:
+    - Nicholas Maselli <maselli.nicholas@gmail.com>
+Implementors: CardanoSharp 7, SaturnNFT, Levvy Finance
```

Comment:
These also need to be a bullet list, formatted like the `Authors`.  From reading the Implementations below I can't be sure if the second 2 are implementations of _your_ algorithm... if there is any subjectivity about that I would include your reference implementation alone for now.